tfhe/src/integer/server_key/radix/tests.rs

```diff
@@ -1165,12 +1176,12 @@ fn integer_smart_scalar_sub(param: ClassicPBSParameters) {
     }
 }
 
-fn integer_unchecked_scalar_decomposition_overflow(param: ClassicPBSParameters) {
-    // This is a regression test. The purpose here is to check if the number of decomposition
-    // blocks doesn't exceed 64 bits. This is why we test only 128 bits size.
-    // If overflow occurs the test case will panic.
+fn integer_unchecked_scalar_decomposition_overflow(param: PBSParameters) {
```

Comment:
this will break the CI as is, unless you want to add multi bit params (which IMO does not make sense for radix non parallelized) here you need to keep ClassicPBSParameters

Reply:
I rebased that too quickly :D